hedera-node/src/main/java/com/hedera/services/store/TypedTokenStore.java

```diff
@@ -315,6 +321,7 @@ private void initModelFields(Token token, MerkleToken immutableToken) {
 		token.setFrozenByDefault(immutableToken.accountsAreFrozenByDefault());
 		token.setType(immutableToken.tokenType());
 		token.setLastUsedSerialNumber(immutableToken.getLastUsedSerialNumber());
+		immutableToken.wipeKey().ifPresent(token::setWipeKey);
```

Comment:
I would say to keep it consistent and remove the `ifPresent`. Always set the `wipeKey` even if it is empty. We can move the `wipeKey` to the other 3 keys as-well (line 318 to 320)